cinema/views.py

```diff
@@ -57,8 +79,14 @@
 ):
     queryset = CinemaHall.objects.all()
     serializer_class = CinemaHallSerializer
-    authentication_classes = (TokenAuthentication,)
-    permission_classes = (IsAdminOrIfAuthenticatedReadOnly,)
+
+    def list(self, request, *args, **kwargs):
+        """Retrieve list of cinema halls"""
+        return super().list(request, *args, **kwargs)
+
+    def create(self, request, *args, **kwargs):
+        """Create a new cinema hall"""
+        return super().create(request, *args, **kwargs)
```

Comment:
Similar to the `list` method, the `create` method in `CinemaHallViewSet` and other similar methods are overriding the default behavior without adding any custom logic. Consider removing these methods if no additional functionality is needed.